test/powershell/Modules/Microsoft.PowerShell.Security/CertificateProvider.Tests.ps1

```diff
@@ -44,22 +44,22 @@ Describe "Certificate Provider tests" -Tags "CI" {
     }
 
     Context "Get-Item tests" {
-        it "Should be able to get a certificate store, path: <path>" -TestCases $testLocations {
+        it "Should be able to get a certificate store, path: <path>" -TestCases $testLocations -Pending {
```

Comment:
Is the test failing because the test is malformed, or because there is an actual problem with code it is testing? If the test is failing due bad test logic, then pending is OK But if the target code for the test is is causing the tail failure, then the test should not be marked pending.

Reply:
I'm going to correct the test logic so it only tests how the code currently works today. Then I will open an issue on the target code. I feel like it is an inconsistency in the target code, but low impact because it is an edge case and is unchanged from 5.1.